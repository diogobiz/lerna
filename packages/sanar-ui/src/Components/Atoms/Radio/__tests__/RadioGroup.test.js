import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESRadioGroup from '../Group'

it('renders correctly', () => {
    const component = (
        <ESRadioGroup>
            <h1>ESRadio</h1>
        </ESRadioGroup>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
