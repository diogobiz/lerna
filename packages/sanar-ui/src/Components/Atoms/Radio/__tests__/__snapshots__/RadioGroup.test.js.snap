// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-radio-group ant-radio-group-outline es-radio-group"
>
  <h1>
    ESRadio
  </h1>
</div>
`;
